apps/docs/content/guides/database/extensions/pgaudit.mdx

```diff
@@ -41,68 +46,389 @@ To disable an extension you can call `drop extension`.
+  );
+```
+
+To prevent pgaudit from monitoring the problematic roles, you'll want to change their `pgaudit.log` values to `none` and `pgaudit.role` values to `empty quotes ''`
+
+```sql
+  -- Use to disable object level logging
+  ALTER ROLE <role name> SET pgaudit.role TO '';
+
+  -- Use to disable global and user level logging
+  ALTER ROLE <role name> SET pgaudit.log TO 'none';
+```
+
+## FAQ
+
+#### How can I use pgaudit to debug database functions?
+
+Technically yes, but it is not the best approach. It is better to check out our [function debugging guide](https://supabase.com/docs/guides/database/functions#general-logging) instead.
+
+#### How can I download database logs?
+
+In the [Logs Dashboard](https://supabase.com/dashboard/project/_/logs/postgres-logs) you can download logs as CSVs. Advanced log drain support is coming soon for Team and above plans.
+
+#### Can I log observed table rows?
+
+pgaudit usually records queries, but not the returned rows. You can modify this behavior with the `pgaudit.log_rows` variable:
+
+```sql
+--enable
+ALTER ROLE postgres SET pgaudit.log_rows TO 'on';
+
+-- disable
+ALTER ROLE postgres SET pgaudit.log_parameter TO 'off';
+```
+
+You should not do this unless you are _absolutely_ certain it is necessary for your use case. It can expose sensitive values to your logs that ideally should not be preserved. Furthermore, it will create redundant logs that will create noise. If done in excess, it can even noticeably impact database performance.
+
+#### Can I log function parameters?
+
+We don't currently support configuring pgaudit.log_parameter because it may log secrets in encrypted columns if you are using pgsodium or Vault.
+
+You can upvote this [feature request](https://github.com/orgs/supabase/discussions/20183) if you'd like this restriction lifted.
+
+#### Why do the official pgaudit docs differ from the Supabase one?
+
+pgaudit allows settings to be applied at 3 database levels:
+
+| Level    | Scope              | Configuration File/Command |
+| -------- | ------------------ | -------------------------- |
+| System   | Entire server      | ALTER SYSTEM commands      |
+| Database | Specific database  | ALTER DATABASE commands    |
+| Role     | Specific user/role | ALTER ROLE commands        |
+
+Supabase limits full privileges for file system and database variables, meaning pgaudit modifications can only occur at the ROLE level. Assigning pgaudit to the 'postgres' role provides it with the role's database visibility, allowing ROLE level adjustments a practical alternative to DATABASE/SYSTEM level configurations.
+
+pgaudit's [official documentation](<(https://www.pgaudit.org)>) focuses on system and database level configs, but its docs officially supports ROLE level configs, too:
+
+> Settings can be specified globally (in postgresql.conf or using ALTER SYSTEM ... SET), at the database level (using ALTER DATABASE ... SET), or at the role level (**using ALTER ROLE ... SET**).
```

Comment:
I don't think we need to inline this quote here

Reply:
This might be better explained at the top in an admonition to make clear this is Supabase's implementation of pgaudit